### DISCUSSION TITLE
Preview Image not working in CuratorPicker

### DISCUSSION THREAD

**Discussion by progressiveautosales1** [awcodes/filament-curator#342] [Q&A]
I am using DO SPACES to host my images. It works on Edit Media page but when you use Curator Picker in other Filament forms. 

This is what it looks like.
![image](https://github.com/awcodes/filament-curator/assets/97930538/c8c54b63-1784-4270-8b96-0a4e6a95ae5f)

If you try to inspect the image tag
The DO_URL is not the path of the URL. Anyone who knows how to set the DO_SPACE URL in the CuratorPicker

src="/curator/base_directory/01-bg-cover-photo-1699302887.jpg?w=1024&h=1024&fit=contain&fm=webp&s=c34a2bf8e35d25b0572f9b73dad30243"

**Comment by awcodes** [comment]
The curator url is a proxy to run the file path through Glide. If you do not need that functionality or your DO space is not set up to allow it then you will need to extend the Media model and provide your own URLs for the thumbnail, medium and large url attributes. 

If your disk and DO space are setup correctly though I would expect it to work. 

Also try adding your DO driver name to the cloud_disks array in the curator config.
